x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/agent_list_page/components/agent_enrollment_flyout/key_selection.tsx

```diff
@@ -62,17 +61,17 @@ function useCreateApiKeyForm(configId: string | null, onSuccess: (keyId: string)
 }
 
 export const APIKeySelection: React.FunctionComponent<Props> = ({ onKeyChange, agentConfigs }) => {
-  const enrollmentAPIKeysRequest = useEnrollmentApiKeys({
-    currentPage: 1,
-    pageSize: 1000,
+  const enrollmentAPIKeysRequest = useGetEnrollmentAPIKeys({
+    page: 1,
+    perPage: 1000,
   });
 
   const [selectedState, setSelectedState] = useState<{
     agentConfigId: string | null;
-    enrollmentAPIKeyId: string | null;
+    enrollmentAPIKeyId: string | undefined;
```

Comment:
Will this key always be present? e.g. `enrollmentAPIKeyId: undefined` vs no `enrollmentAPIKeyId`

I ask because I expected `enrollmentAPIKeyId?: string`

Reply:
Going to use optional here 👍